### ISSUE TITLE
When MTA connects to peer, do rDNS lookup

### ISSUE THREAD

**Issue by poolpOrg** [OpenSMTPD/OpenSMTPD#890]
Similarly to #889 we need both forward and rDNS before calling report_link_connect

**Comment by ngortheone** [comment]
Is this somehow related to https://github.com/OpenSMTPD/OpenSMTPD/issues/968 ?

**Comment by poolpOrg** [comment]
Nope, there's no rDNS lookup done in the MTA at this point, this is something we want to implement.
